cerbero/build/fridge.py

```diff
@@ -92,9 +108,37 @@ def generate_binary(self, recipe):
         p.pre_package()
         paths = tar.pack(self.binaries, devel=True, force=True, force_empty=True)
         p.post_package(paths)
+        # Replace every occurence of the prefix with a variable for later replacing
+        for filename in paths:
```

Comment:
Can't we integrate this as an option in the DistTarball so we avoid compressing/extracting/compression